### ISSUE TITLE
Configuration: expire_interval for snapshots

### ISSUE THREAD

**Issue by c4talyst** [blakeblackshear/frigate#14002]
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to configure the `expire_interval` of `snapshots` so that Frigate can reduce disk activity.  

**Describe the solution you'd like**
In configuration, there is a `expire_interval` for `record`; there is no such setting for `snapshots`. If I could set the interval for `snapshots` I could reduce disk activity. 

The default appears to be 300 seconds (every 5 minutes)
https://github.com/blakeblackshear/frigate/blob/master/frigate/storage.py#L213

**Describe alternatives you've considered**
I could move snapshots/clips to a different disk (e.g. a SSD) to prevent disks from spinning

**Additional context**
Debugging this as per https://github.com/blakeblackshear/frigate/discussions/10611#discussioncomment-10772586 and using `inotifywait` to identify what was causing the disk to spin; command below: 

```
inotifywait -m -r /mnt/disk2/cctv | awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0; fflush() }'
```

**Comment by jmot205** [comment]
This would be a great addition so we can cut down on IO and let HDDs stay spun down for much longer

**Comment by NickM-27** [comment]
To be clear you can already do this by mapping the clips directory separate from recordings

**Comment by jmot205** [comment]
Yeah that's definitely an option I've been considering, it's just that HDD space is at less of a premium for my system currently.

**Comment by c4talyst** [comment]
> To be clear you can already do this by mapping the clips directory separate from recordings

Correct - and as per the description of the issue - this is the alternative currently in use to reduce spinning disk activity. However, having it configurable similar to recording would be the goal, as @jmot205 mentions, SSD space is a premium. 

Thanks for a great app

**Comment by WaffleMaster22** [comment]
> To be clear you can already do this by mapping the clips directory separate from recordings

How would I do this? I can't find anything about that in the docs. I added a new path to the docker container but idk what to put in the config. 'path: /clips/path' does not work under snapshots

**Comment by NickM-27** [comment]
you don't define anything in the frigate config.

**Comment by c4talyst** [comment]
> > you don't define anything in the frigate config.
> 
> Oh okay, the way unraid has the paths set up by default made 'media' all one folder with clips, recordings, and exports. But I figured out how to separate it, thanks

Can you share, to help the next person who stumbles across this post. TIA

**Comment by sevenmp** [comment]
> > you don't define anything in the frigate config.
> 
> Oh okay, the way unraid has the paths set up by default made 'media' all one folder with clips, recordings, and exports. But I figured out how to separate it, thanks

Can you explain how did you achieve this in unraid? How did you split the mapping of snapshots from the clips?

**Comment by c4talyst** [comment]
> > > you don't define anything in the frigate config.
> > 
> > 
> > Oh okay, the way unraid has the paths set up by default made 'media' all one folder with clips, recordings, and exports. But I figured out how to separate it, thanks
> 
> Can you explain how did you achieve this in unraid? How did you split the mapping of snapshots from the clips?

No idea where this is configured in frigate? Can't find it on the "full reference config" either https://docs.frigate.video/configuration/reference/

**Comment by NickM-27** [comment]
It's already split. You just define one folder in unraid to map to /media/frigate/clips and another to /media/frigate/recordings

**Comment by EcN000** [comment]
Any updates on this? Unraid doesn't seem to have the option to map separate directories, or I don't know how. 

https://imgur.com/a/QNmH880

**Comment by NickM-27** [comment]
Unraid does have this option, I do this in Unraid.

**Comment by EcN000** [comment]
I think I figured it out, delete the original path and split into 3: https://imgur.com/a/GHhLJup

**Comment by EcN000** [comment]
I think I figured it out. You remove the original mapping, then manually add 3 new paths: https://imgur.com/a/GHhLJup

Then you can have exports and the recordings path continue to move to array, while the clips can stay in cache. @c4talyst @sevenmp

**Comment by NickM-27** [comment]
Yep that's right, nice work

**Comment by EcN000** [comment]
Was the deletion of the default path necessary? Could unraid have been smart enough to see a nested path mapping to keep clips folder in cache?

**Comment by NickM-27** [comment]
To be clear it's a docker function not Unraid, but in general it's best to not have redundant mappings

**Comment by EcN000** [comment]
Got it, thank you

**Comment by c4talyst** [comment]
Thank for sharing your config @EcN000 - I will give that a go 🙏 

I think the original feature request is still valid, having this period configurable would be ideal. Thanks for a great project.
